src/store/modules/arrow/mutations.ts

```diff
@@ -0,0 +1,11 @@
+import { MutationTree } from 'vuex'
+import { EMutationArrow, IStateArrow, Mutations } from '@/store/modules/arrow/typesArrow'
+
+export const mutations: MutationTree<IStateArrow> & Mutations = {
+  [EMutationArrow.SET_DATA]: (state, payload) => {
+    state.arrowsData?.push(payload)
```

Comment:
why `?`

Reply:
Thanks. this is not needed here. Will fix it in next commit